components/Tabs/tests/unit/MDCItemBarCellTests.m

```diff
@@ -59,4 +59,29 @@ - (void)testTitleNumberOfLines {
   XCTAssertEqual(cellWithTextOnly.titleLabel.numberOfLines, 2);
 }
 
+/// Tests that a cell that was initially configured as image-only style, and then changed to
+/// image-and-title style, will result in the correct title text.
+- (void)testTitleAfterStyleChange {
+  MDCItemBarStyle *iconOnlyStyle = [[MDCItemBarStyle alloc] init];
+  iconOnlyStyle.shouldDisplayImage = YES;
+  iconOnlyStyle.shouldDisplayTitle = NO;
+
+  MDCItemBarStyle *iconAndTextStyle = [[MDCItemBarStyle alloc] init];
+  iconAndTextStyle.shouldDisplayImage = YES;
+  iconAndTextStyle.shouldDisplayTitle = YES;
+
+  // Create a cell and set the style before settting the image/title. That is the order items will
+  // be configured in the app runtime.
+  MDCItemBarCell *cell = [[MDCItemBarCell alloc] initWithFrame:CGRectZero];
+  [cell applyStyle:iconOnlyStyle];
+  [cell setImage:[UIImage imageNamed:@"TabBarDemo_ic_info"]];
+  [cell setTitle:@"A title"];
```

Comment:
Super-nit: Prefer invoking properties if they have been defined.
cell.image = [UIImage imageNamed:@"info"];
cell.title = @"A title";

There is an automated tool under Xcode -> Edit -> Convert -> Modern Obj-C Syntax

Reply:
Done.